index.js

```diff
@@ -67,7 +72,7 @@ var publicAgent = {
     var logger = new Logger(config.logLevel, '@google/cloud-trace');
```

Comment:
It is not used anywhere else, and is a very descriptive value. It is better left inline here.